docs/windows.md

```diff
@@ -1,5 +1,50 @@
 # Building on Windows
 
+## Install Git For Windows
+
+As Git is used by the Go language to download dependencies, you need
+to install Git, for example from http://git-scm.com/.
+
+You also must make sure that the directory that contains the Git For
+Windows binary is in the Path environment variable. Note that Git For
+Windows has a 'git' binary in a 'Git\bin' directory and another one in
+a 'Git\cmd' directory. You should only put the 'Git\cmd' directory in
+the Path environment variable.
+
+## Install Go
+
+Please install the Go language as explained on
+https://golang.org/doc/install.
+
+You must make sure that the GOROOT environment variable is set and
+that the %GOROOT%/bin directory is in the Path environment variable.
+
+The GOPATH environment variable should also be set to a directory that
+you have created, and the %GOPATH/bin directory should also be in the
+Path environment variable.
+
+## Download go-ipfs and its dependencies
+
+The following commands should download or update go-ipfs and its
+dependencies:
+
+```sh
+go get -u github.com/whyrusleeping/gx
+go get -u github.com/whyrusleeping/gx-go
+go get -u github.com/ipfs/go-ipfs
+cd %GOPATH%/src/github.com/ipfs/go-ipfs
+gx --verbose install --global
+```
```

Comment:
The problem is that installing "make" on Windows is usually done by installing Cygwin which takes a lot of time. We should perhaps check if there is a way to install make without Cygwin but it might be for another PR.